recipes/epanet-dev/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "epanet-dev" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/OpenWaterAnalytics/epanet-dev.git
+  git_tag: master
+  patches:
+    # 
+    - macos.patch
+
+build:
+  skip: True  # [win and py27 or win and py34]
+  number: 0
+  features:
+    - vc14  # [win and py35]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake >=2.8
```

Comment:
This needs to be `>=3.4` for Windows. That's because `epanet-dev` is using `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS`, which was introduced in 3.4:

https://blog.kitware.com/create-dlls-on-windows-without-declspec-using-new-cmake-export-all-feature/